ecommers/lib/core/models/data_models/category.dart

```diff
@@ -6,9 +7,21 @@ part 'category.g.dart';
 class Category {
   final int id;
   final String title;
+  final Categories type;
+  final String color1;
```

Comment:
color1 isn't informative, change to labelColor, color2 change to iconColor

Reply:
these two colors are gradient colors